packages/victory-bar/src/geometry-helper-methods.js

```diff
@@ -1,12 +1,13 @@
 /**
  * A point in the 2d plane
+ * @constructor
  * @param {number} x - x coordinate
  * @param {number} y - y coordinate
  */
-function Point(x, y) {
-  this.x = x;
-  this.y = y;
-}
+const Point = function (x, y) {
```

Comment:
Writing `function Point(x, y)` results in ESLint error `func-style`
Writing `const Point = function (x, y)` results in ESLint error `no-invalid-this` 
Which is preferred?

Reply:
`const Point = (x, y) => ...` should pass our linting rules. Otherwise, prefer `function Point(x, y)` and add an exception for `func-style`